kuma/users/jinja2/users/includes/delete-user-content.html

```diff
@@ -7,7 +7,20 @@
             <p>{{ _('Deleting your account loses any preferences you have set.') }}</p>
             <p>{{ _('Although we can delete your account, any revisions you have made to pages on MDN will remain. Please select how would like us to handle the attribution for your changes.') }}</p>
             {{ form.attributions.errors }}
-            {{ form.attributions }}
+            <ul>
+                {% for field in form.attributions %}
+                    <li>
+                        <label for="{{ field.id_for_label }}">
+                            {{ field.tag() }}
+                            {% if "Anonymous" not in field.choice_label %}
```

Comment:
This feels a bit brittle. I wonder if there's a way we could do this inside `kuma/users/forms.py` instead.

It's been years since I've really worked with Django forms; I'll re-read the docs and see.

Reply:
It does for sure. Currently, we do not have the context to know who the user is within the Form class. If we could do it that way, that would definitely be much better.

Reply:
Maybe the approach in #6523 is better, in that case? 🤷‍♂